Dockerfile

```
@@ -1,10 +1,13 @@
-FROM node:18
+# Build stage
+FROM node:18 AS build
 WORKDIR /app
 COPY package*.json ./
+RUN npm ci
 COPY . .
-RUN npm install
+RUN npm run build
 
-ENV PORT = 8080
-EXPOSE 8080
-
-CMD [ "npm", "start" ]
\ No newline at end of file
+# Production stage
+FROM nginx:stable-alpine
+COPY --from=build /app/dist /usr/share/nginx/html
+EXPOSE 80
+CMD ["nginx", "-g", "daemon off;"]
```

nit: I don't think the semicolon is required
```suggestion
CMD ["nginx", "-g", "daemon off"]
```